lib/datasource/terraform-module/index.ts

```diff
@@ -97,8 +98,6 @@ export async function getReleases({
       dep.homepage = `https://registry.terraform.io/modules/${repository}`;
     }
     logger.trace({ dep }, 'dep');
-    const cacheMinutes = 30;
-    await packageCache.set(cacheNamespace, pkgUrl, dep, cacheMinutes);
```

Comment:
why caching is removed?

Reply:
Caching has been moved out of the datasources, therefore the internal caching here is not needed anymore. https://github.com/renovatebot/renovate/blob/master/lib/datasource/index.ts#L218

Reply:
Actually that's just a mem cache for per-repo caching, to make sure we don't waste work fetching and parsing the same package more than once. It's cleared at the start of every repo. We still want the package cache to store results across runs for 30 minutes so that it is shared across repos and runs.

Reply:
@rarkins Ah I see, then I revert this.  The caching for `terraform-provider` has been removed with  #6513. Do you want a separate PR for this?